### PR TITLE
PUB-963 - Account Home page

### PR DESCRIPTION
**Pull request by commitBlob**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-963

### Change description ###
- implement account-home page with useful links (in the future user will be redirected to this page after successful login)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
yourAccount

### CODE REVIEW

**Review by furnivall — DISMISSED** [review]
Hi, looks good man :)

**Review by ChrisS1512 — APPROVED** [review]

**Review by furnivall — APPROVED** [review]
looks good

**Review by furnivall — DISMISSED** [review]

**Review by alberto-hortiguela — DISMISSED** [review]
LGTM

**Review by furnivall — APPROVED** [review]
I'll be sure to renew my TB license immediately :D

File: src/main/views/account-home.njk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            <div class="govuk-!-margin-top-2 govuk-!-margin-bottom-2">
+                <div class="account-card govuk-!-margin-left-2 govuk-!-margin-right-2">
+                    <p>
+                        <a href="{{ card.link }}"
+                           class="govuk-link govuk-body govuk-!-font-weight-bold">{{ card.title }}</a>
+                    </p>
+                    <p class="govuk-body">{{ card.description }}</p>
+                </div>
+            </div>
+        {% endfor %}
```
Comment (kurt-uk) [inline src/main/views/account-home.njk:26]: this is nice and neat!

**Review by kurt-uk — APPROVED** [review]

File: src/main/views/account-home.njk (hunk 1 above)
Comment (commitBlob) [reply src/main/views/account-home.njk:26]: Thanks,
resolving as it is blocking the merge 😂

**Pull request merged**